app/views/layouts/embed.html.erb

```diff
@@ -37,9 +37,16 @@
           }
         }
 
-        window.onload = function() {
-          // Send a post message with our loaded height
-          postUp({type: 'discourse-resize', height: document['body'].offsetHeight});
+	window.onload = function() {
+	  // get state info from data attribute
+	  var header = document.querySelector('header');
+	  var state = 'unknown';
+	  if (header) {
+	    var state = header.getAttribute('data-embed-state');
```

Comment:
You are redeclaring `state` here. No `var` keyword should be present.

Reply:
ok, I'll fix this.